htcanalyze/htcanalyze.py

```diff
@@ -14,9 +14,12 @@
 from rich.progress import Progress, track
 from typing import List
 
+# import own class
+from htcanalyze.resource import Resource, create_avg_on_resources
+
 # typing identities
-log_inf_list = List[dict]
-list_of_logs = List[str]
+LogDataList = List[dict]
```

Comment:
stick to snake case for this, so `log_data_list`. I'm still not a fan of redeclaring object types though ;)

Reply:
Got it, I think in this case I can get rid of this definition, as it is not complicated.